src/main.cpp

```diff
@@ -1,21 +1,38 @@
+#include <QtDebug>
 #include <QApplication>
 #include <QStringListModel>
 #include <QStandardItemModel>
+#include <QAbstractItemModelTester>
+
+#include "recentproject.h"
+#include "programsettings.h"
 #include "projectviewerwindow.h"
+#include "recentprojectsmodel.h"
+
 
 int main(int argc, char *argv[])
 {
     QApplication a(argc, argv);
 
-    QStandardItemModel model;
+    RecentProject pr1("/home/game1");
+    RecentProject pr2("/home/igor/bla/blabla/game2");
+    RecentProject pr3("/home/igor/Documents/game3");
+
+    QVector<RecentProject> projects{pr1,pr2,pr3};
+
+    ProgramSettings settings{projects};
+
+    RecentProjectsModel model(settings);
+
+//    QStandardItemModel model;
```

Comment:
Remove commented code. Code like that calls "dead code", "Zombie code" or "Ghost code" :)